packages/web/src/pages/sign-up-page/components/AccountHeader.tsx

```diff
@@ -0,0 +1,87 @@
+import { Avatar, Box, Flex, Text } from '@audius/harmony'
+
+import {
+  getCoverPhotoField,
+  getHandleField,
+  getNameField,
+  getProfileImageField
+} from 'common/store/pages/signon/selectors'
+import { useMedia } from 'hooks/useMedia'
+import { useSelector } from 'utils/reducer'
+
+export const AccountHeader = () => {
+  const coverPhoto = useSelector(getCoverPhotoField)
+  const profileImage = useSelector(getProfileImageField)
+  const { value: displayName } = useSelector(getNameField)
+  const { value: handle } = useSelector(getHandleField)
+
+  const { isMobile } = useMedia()
+
+  const avatarSize = isMobile ? 72 : 120
+
+  return (
+    <Box>
+      <Box h={isMobile ? 96 : 168} css={{ overflow: 'hidden' }}>
+        <Box
+          h='100%'
+          w='100%'
+          border='default'
+          css={{
+            backgroundImage: `linear-gradient(0deg, rgba(0, 0, 0, 0.20) 0%, rgba(0, 0, 0, 0.00) 100%), url(${coverPhoto.url}), url(${coverPhoto.url})`,
+            backgroundPosition: '2.033px -355.342px, center, center',
+            backgroundSize: '100% 571.429%, cover',
```

Comment:
Sizes look weird. How do we come up with this? Are all cover photos the same size?

Reply:
all from the design, agreed these are quiteee random. will follow up with sammie on what she thinks is best here

Reply:
@dylanjeffers what sizes are we referring to here?

Reply:
Can chat more tomorrow but the various values like image positioning, and all the numbers in this code block. I basically copied straight from design so wanted to confirm these are the exact values we want or were just specific to the image you used